benchmark/src/main/java/ai/timefold/solver/benchmark/impl/report/LineChart.java

```diff
@@ -173,7 +187,7 @@ public void writeToFile(Path parentFolder) {
     }
 
     public static final class Builder<X extends Number & Comparable<X>, Y extends Number & Comparable<Y>> {
-
+        private static final int MAX_CHART_WIDTH = 1280;
```

Comment:
Seems low, no? I'd increase it up to 4K width, which is 3840.

Reply:
Yes, we can increase it. Should we also change the chart setting?

Reply:
I'd keep it what it is - but when a user scales it up manually, they should get all the extra points. Therefore changing the maximum constant.